atd-vze/src/views/Crashes/Maps/CrashEditCoordsMap.js

```diff
@@ -31,8 +31,8 @@ const CrashEditCoordsMap = ({
   const mapRef = React.useRef();
   const [isDragging, setIsDragging] = React.useState(false);
   const [markerCoordinates, setMarkerCoordinates] = React.useState({
-    latitude: latitude_primary,
-    longitude: longitude_primary,
+    latitude: latitude_primary || defaultInitialState.latitude,
+    longitude: longitude_primary || defaultInitialState.longitude,
```

Comment:
Another small fix so that the marker has coordinates when the crash does not. Previously, the marker would only appear after moving the map since it is captured on a map drag event. Now, the initial location of the marker is the same lat/lon used to initialize the map when adding coordinates to a crash that has none.

Reply:
ignore this too - caused another bug 🐛